templates/ai/what-is-kubeflow.html

```diff
@@ -20,7 +17,7 @@ <h1 class="u-sv2">What is Kubeflow?</h1>
           <a class="p-button--positive" href="https://charmed-kubeflow.io/docs/install">
           Charmed Kubeflow
           </a>
-          <a href="https://charmed-kubeflow.io/#get-in-touch">Contact us now&nbsp;&rsaquo;</a>
```

Comment:
I think spaces probably don't have any problems rendering. Just the `&rsaquo;`

Reply:
if you don't add a `&nbsp;` it is possible that the  `&rsaquo;` can fall onto the next line on certain screen sizes